### PR TITLE
Spawn monster to game_scene

### PR DESCRIPTION
**Pull request by ligamesh123**

### CODE REVIEW

**Comment by ra1nns** [comment]
Very nice keep in mind we have 2 mobs try to make it so both types of mobs can spawn instead of just one spawning in

**Comment by Tobeast1** [comment]
In full screen the mobs spawn in view of the player. Try changing the Mode and Aspect in Project>Project Settings>Display>Window.

File: game.tscn [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
-position = Vector2(130, 153)
+position = Vector2(1307, 738)
+
+[node name="Path2D" type="Path2D" parent="Player"]
+curve = SubResource("Curve2D_avoj2")
+
+[node name="PathFollow2D" type="PathFollow2D" parent="Player/Path2D"]
+unique_name_in_owner = true
+position = Vector2(-876, -546)
+rotation = -0.00174927
```
Comment (ExclusiveKnight) [inline game.tscn:25]: This PR works like it should, but PathFollow2D shouldn't be rotated. Reset it's rotation.

**Review by ExclusiveKnight — CHANGES_REQUESTED** [review]

**Comment by DNQ4** [comment]
I think we should also make it so that the game keeps track of how many enimes there is? Or at least make the enemies slow in the beginning of the game, I do not know we if we are going to implement player leveling and new weapons upgrades during the game, but it could end up that the mobs overwhelm the player in the beginning of the game with their speed and spawn rate.

**Comment by ligamesh123** [comment]
Right now we have 2 options to spawn different monster:
1. We can spawn them same time in one function .
2. We can have a delay between different enemy spawn time.
Or if u guy have better option so we can do this.

**Comment by xix-xeaon** [comment]
Closing in favor of #27

**Pull request closed without merging**